pkg/controller/cloud/node_controller.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/client-go/tools/record"
 	clientretry "k8s.io/client-go/util/retry"
 	cloudprovider "k8s.io/cloud-provider"
+	utilnode "k8s.io/cloud-provider/node/helpers"
```

Comment:
`cloudnodeutil`? having `utilnode` and `nodeutil` imports is a little o_O

Reply:
Indeed, renamed